src/module/item/abc/data.ts

```diff
@@ -1,4 +1,7 @@
+import type { ImageFilePath } from "@common/constants.mjs";
+import type { ItemUUID } from "@common/documents/_module.mjs";
```

Comment:
```suggestion
import type { ImageFilePath } from "@common/constants.d.mts";
import type { ItemUUID } from "@common/documents/_module.d.mts";
```

Reply:
Done but we need a way to inform the editor to pick the right import somehow